drivers/volume/kdmp/kdmp.go

```diff
@@ -371,6 +375,24 @@ func (k *kdmp) GetBackupStatus(backup *storkapi.ApplicationBackup) ([]*storkapi.
 	}
 	return volumeInfos, nil
 }
+
+// Check whether to run kopia executor in debug mode
```

Comment:
If its check function, then the function name is missleading. enableKopiaExecutorDebugMode() means enabling it not checking.
Suggestion:  IsKopiaExecutorDebugModeEnabled()